framework/db/ActiveRelationTrait.php

```diff
@@ -23,6 +23,12 @@
  */
 trait ActiveRelationTrait
 {
+    /**
+     * @var bool whether use strtolower() for keys while populate relation.
+     * For example, mysql can join records by strings with different cases (Key = key), but AR doesn't
```

Comment:
Example isn't clear. Would you please provide an example of what is meant in order to decide on better text?

Reply:
The example is available here https://github.com/yiisoft/active-record/issues/22